server/datastore/mysql/software.go

```diff
@@ -1915,7 +1915,7 @@ func (ds *Datastore) ListHostSoftware(ctx context.Context, hostID uint, includeA
 				INNER JOIN
 					software s ON hs.software_id = s.id
 				WHERE
-					hs.host_id = :host_id AND
+					hs.host_id = ? AND
```

Comment:
not super critical but, any reason to change this? I know we're doing a new `IN` above, but `sqlx.Named` is being called first and should replace the named parameters with `?`, was it complaining?

Reply:
Yeah, I was having trouble making it work smoothly so I opted for this approach.